Rakefile

```diff
@@ -29,6 +29,11 @@ else
   task :build do; end
 end
 
+task :populate_proxy do
+  require 'toxiproxy'
+  Toxiproxy.populate(File.expand_path('../test/fixtures/toxiproxy.json', __FILE__))
+end
```

Comment:
Interesting, I've always done this in the test helper. This is not a bad idea, but the problem is that running a single test file (which I for example do often) will fail with this approach. Since Semian will have a bunch of latency tests that will be prone to being slow, it's likely we'll end up in a situation where developers will run single tests a lot—where this won't work.